native_client/setup.py

```diff
@@ -17,6 +17,9 @@
     numpy_include = ''
     assert 'NUMPY_INCLUDE' in os.environ
 
+def read(fname):
+    return open(os.path.join(os.path.dirname(__file__), fname)).read()
```

Comment:
This is meant to handle cases where setup.py is called from a different directory, right?

Reply:
Yep. It's even what they use in the doc